dist/Icon.d.ts

```diff
@@ -1,13 +1,13 @@
 /// <reference types="react" />
+import { SpectrumComponentSize } from './common';
 declare namespace Spectrum {
     type IconName = 'ui:AlertMedium' | 'ui:AlertSmall' | 'ui:ArrowDownSmall' | 'ui:ArrowLeftMedium' | 'ui:ArrowUpSmall' | 'ui:Asterisk' | 'ui:CheckmarkMedium' | 'ui:CheckmarkSmall' | 'ui:ChevronDownMedium' | 'ui:ChevronDownSmall' | 'ui:ChevronLeftLarge' | 'ui:ChevronLeftMedium' | 'ui:ChevronRightLarge' | 'ui:ChevronRightMedium' | 'ui:ChevronRightSmall' | 'ui:ChevronUpSmall' | 'ui:CornerTriangle' | 'ui:CrossLarge' | 'ui:CrossMedium' | 'ui:CrossSmall' | 'ui:DashSmall' | 'ui:DoubleGripper' | 'ui:FolderBreadcrumb' | 'ui:HelpMedium' | 'ui:HelpSmall' | 'ui:InfoMedium' | 'ui:InfoSmall' | 'ui:Magnifier' | 'ui:More' | 'ui:SkipLeft' | 'ui:SkipRight' | 'ui:Star' | 'ui:StarOutline' | 'ui:SuccessMedium' | 'ui:SuccessSmall' | 'ui:TripleGripper';
-    type IconSize = 'xxs' | 'xs' | 's' | 'm' | 'l' | 'xl' | 'xxl';
```

Comment:
Do we lose icon sizes by this being replaced with common?
Do more icon sizes exist than do common sizes?

Reply:
Somebody in Slack reported this as a "bug". I did not test it but I blindly trust it. :-D